opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py

```diff
@@ -57,6 +58,13 @@
 _ENV_VALUE_UNSET = ""
 
 
+class LogDroppedAttributesWarning(UserWarning):
```

Comment:
What do you think about adding a comment here to touch on the motivation behind this class?

Reply:
I think it makes sense. Thanks for your remark!